src/sentry/monitors/tasks.py

```diff
@@ -391,3 +391,55 @@ def mark_checkin_timeout(checkin_id: int, ts: datetime):
         )
 
         mark_failed(checkin, ts=most_recent_expected_ts)
+
+
+@instrumented_task(
+    name="sentry.monitors.tasks.mark_checkin_timeout",
+    max_retries=0,
+    record_timing=True,
+)
+def mark_checkin_timeout_verbose(checkin_id: int, ts: datetime, **kwargs):
```

Comment:
I would just add this to the original one. adding kwargs isn't going to break anything.

Then you can just log the kwargs for now

Reply:
if you promise haha